src/class-admin-notices.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Duplicate Post class to manage the admin notices.
+ *
+ * @package Duplicate_Post
+ */
+
+namespace Yoast\WP\Duplicate_Post;
+
+/**
+ * Represents the Admin_Notices class.
+ */
+class Admin_Notices {
+
+	/**
+	 * Initializes the class.
+	 */
+	public function __construct() {
+		$this->register_hooks();
+	}
+
+	/**
+	 * Adds hooks to integrate with WordPress.
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+		\add_filter( 'removable_query_args', [ $this, 'add_removable_query_args' ] );
+		\add_action( 'admin_notices', [ $this, 'single_action_admin_notice' ] );
+		\add_action( 'admin_notices', [ $this, 'bulk_action_admin_notice' ] );
+	}
+
+	/**
+	 * Adds 'rewriting' to the removable query args.
+	 *
+	 * @ignore
```

Comment:
Question due to my ignorance: what is the purpose of this `@ignore` notation?

Reply:
It is meant to skip generation of the documentation for that element. Actually this is a leftover from a copied snippet (there was a plan to generate full automatic docs for the API), I'll remove it.